ci/conform-nvim.nix

```diff
@@ -0,0 +1,19 @@
+{
+  vimPlugins,
+  lib,
+  writeText,
+}:
+let
+  formattersDir = "${vimPlugins.conform-nvim}/lua/conform/formatters";
+  formatterFiles = builtins.attrNames (builtins.readDir formattersDir);
+  supportedFormatters =
+    let
+      luaFiles = builtins.filter (lib.hasSuffix ".lua") formatterFiles;
+    in
+    map (lib.removeSuffix ".lua") luaFiles;
+in
+writeText "conform-formatters.nix" (
+  "# WARNING: DO NOT EDIT\n"
+  + "# This file is generated with packages.<system>.conform-formatters, which is run automatically by CI\n"
+  + (lib.generators.toPretty { } supportedFormatters)
```

Comment:
I know we've done this in the past, but I'd prefer to avoid generating nix code. Generating an actual data format such as CSV, txt lines, JSON, TOML, YAML, etc is much more appropriate.